style.css

```diff
@@ -1014,3 +1019,235 @@ button:active {
   padding-top: 20px;
 }
+
+/* media query */
+@media screen and (min-width: 768px) {
+  .main-mobile {
+    display: flex;
+    width: auto;
+  }
+
+  .nav-links ul li {
+    display: inline;
+    color: #36b37e;
+    flex-direction: row;
+    justify-content: space-between;
+    justify-items: center;
+    left: 130%;
+  }
+
+  #menu {
+    display: none;
+  }
+
+  #welcome {
+    display: none;
+  }
+
+  .headline {
+    background-image: url(/images/Header-llustration-desktop@2x.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    position: absolute;
+    width: 100%;
+  }
+
+  .black {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+    margin-top: -5%;
+  }
+
+  .text {
+    width: 720px;
+    height: 96px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+    margin-top: -8%;
+  }
+
+  #text2 {
+    width: 947px;
+    margin-top: -5%;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .card-workrow {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 0;
+    margin-left: 5px;
+    width: 1171px;
+    height: 988px;
+  }
+
+  .about-me {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 80px 0;
+    gap: 48px;
+    position: absolute;
+    width: 1171px;
+    height: 808px;
+    left: 235px;
+    top: 2331px;
+  }
+
+  .about-text {
+    width: 100%;
+    flex-direction: row;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 0;
+    margin-top: -450px;
+    margin-left: -50%;
+    background-image: url(/images/about.png), url(/images/aboutright.png);
+    background-repeat: no-repeat, no-repeat;
+    background-position-x: left, right;
+    background-position-y: center, top;
+    background-size: 30%, 10%;
+  }
+
+  .p-text {
+    flex-direction: row;
+    width: 920px;
+    height: 96px;
+    margin-left: 50%;
+  }
+
+  .s-text {
+    width: 720px;
+    height: 96px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+    margin-left: 30%;
+  }
+
+  #resume-button {
+    margin-left: 650px;
+  }
+
+  .about-card {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 8px;
+    gap: 24px;
+    width: 1071px;
+    height: 360px;
+    margin-left: -45%;
+  }
```

Comment:
- [ ] As suggested by the previous reviewer, please try to align the Technologies cards in the center. As I mentioned you can leverage the power of flexbox. Please try not to set fixed `width` and height on all your elements because they are making your design look bad. As you can see, there is a lot of layout issues that do not align with the Figma design given. In order to create a responsive design, we should not set these property values to a fixed number because it will be difficult to handle all the screen sizes. 
**HINT** 
```
You may wrap all these Technology cards in the div
Then add a `display flex` to that div
`justify-content: center`
`align-items: center`
```
**Your Design**
![](https://user-images.githubusercontent.com/33752735/186698650-94811356-1896-4588-9e69-2937bbed0b29.png)

**Figma**
![](https://user-images.githubusercontent.com/33752735/186698894-4ee3b599-84f7-4a34-94f8-2d44cfba56ba.png)